base/stream.jl

```diff
@@ -278,8 +278,14 @@ function wait_readbyte(x::LibuvStream, c::UInt8)
             wait(x.readnotify)
         end
     finally
-        if isempty(x.readnotify.waitq)
-            stop_reading(x) # stop reading iff there are currently no other read clients of the stream
+        if JULIA_PARTR
+            if isempty(x.readnotify)
+                stop_reading(x) # stop reading iff there are currently no other read clients of the stream
+            end
+        else
```

Comment:
This `else` and some others are missing the comments saying `## ! JULIA_PARTR`.
I had got used to using those to navigate, they are helpful.